src/orm/axios.js

```diff
@@ -5,11 +5,12 @@ export default class Axios {
     this.instance = http.axios || axios.create(http);
     this.setAuthentication(http.access_token);
 
-    this.instance.interceptors.response.use(
-      response => http.onResponse(response, this.instance),
-      error => http.onError(error, this.instance),
-    );
-
+    if (! http.axios || this.instance.interceptors.response.handlers.length < 1) {
```

Comment:
This code in question is highly unsafe. If the external instance contains any handlers already this will not run. It would be better to let this always run but to use axios eject method when this instance is destroyed.

Check https://github.com/axios/axios#interceptors for reference on how to use eject.